CHANGELOG.md

```diff
@@ -589,3 +589,4 @@
 [@natematykiewicz]: https://github.com/natematykiewicz
 [@lulalala]: https://github.com/lulalala
 [@gmcgibbon]: https://github.com/gmcgibbon
+[@tk0miya]: https://github.com/tk0miya
```

Comment:
Can you remove it? Because it will be auto-generated by the release task.

Reply:
Okay, I misunderstood.